aspnetcore/fundamentals/servers/index.md

```diff
@@ -153,6 +153,9 @@ When launching an app from a command prompt in the project's folder, [dotnet run
   * Public-facing edge server connections use HTTP/2, but the reverse proxy connection to Kestrel uses HTTP/1.1.
   * Target framework: Not applicable to IIS out-of-process deployments.
 
+> [!NOTE]
```

Comment:
You should update the kestrel doc as well.
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-2.2#http/2-support